src/main/java/ChatResponseRespone.java

```diff
@@ -0,0 +1,29 @@
+import lombok.Data;
+
+import java.util.List;
+
+@Data
+public class ChatResponseRespone {
```

Comment:
The @Data annotation includes @Getter, @Setter, @ToString, and @EqualsAndHashCode. Using all of them together may violate the Single Responsibility Principle by combining multiple responsibilities in one class.